### ISSUE TITLE
Login and Reset your password have non-descript heading hidden from visual users

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#33]
URL: https://hcdev.openplus.ca/en/user/password and Login page
User Agent: Chrome
Language: both

**Issue Description**: "Primary Tabs" Heading level 2 is being announced to screen reader users. This text is visually hidden from users and the text conveys nothing to users.

**Steps to reproduce**: 
Run WAVE and turn off styles.
Heading 2 = Primary Tabs is displayed. 
View code -> heading is visually hidden 

**Expected results**: This conveys information and should be available to all users (aka visible) Headings should be descriptive. "Primary Tabs" does not convey what is below which is the navigation.

**Comment by davidgotlieb** [comment]
Reported to maintainer of module
